src/main/java/TaskList.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Storage for user inputs up to 100 records.
+ *
+ * @author JamesLiuZX
+ * AY2223-S2 CS2103T
+ */
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+public class TaskList {
+    //Variables are kept protected and accessed only through internal getter and setter methods.
+    protected static final int size = 100;
+    protected ArrayList<Task> records;
+    private final String filePath = "./data/duke.txt";
+    private final String database = "duke.txt";
+    private Path path = Paths.get(filePath);
+    private DatabaseWriter dw = new DatabaseWriter(path);
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yy HH:mm");
+
+    //Constructor
+    public TaskList() {
+        records = new ArrayList<>(size);
+    }
+
+    public void mark(int index) {
```

Comment:
Remember to add javadocs for all the public function

Reply:
Noted.